filesystem.go

```diff
@@ -161,7 +161,7 @@ func writeable(path string) bool {
 func mustMakeDirectory(path string) error {
```

Comment:
The `must*` prefix in Go is by convention used for functions that panic if they fail. If they return an `error`, the convention would just be `makeDirectory`.